packages/zero-cache/src/config/config.test.ts

```diff
@@ -9,6 +9,9 @@ import {
   type Options,
 } from './config.js';
 
+// For some reason, Typescript doesn't figure out `import ansis from 'ansis'`
+const ansis = new Ansis();
```

Comment:
> When I do:
> 
> ```ts
> import ansis from 'ansis';
> ```
> 
> I get the following typescript error:
> 
> <img alt="Screenshot 2024-11-04 at 11 56 29" width="974" src="https://private-user-images.githubusercontent.com/132324914/382900369-6118a8cc-622f-4749-ac53-84400a2bc200.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUyOTU4NDIsIm5iZiI6MTczNTI5NTU0MiwicGF0aCI6Ii8xMzIzMjQ5MTQvMzgyOTAwMzY5LTYxMThhOGNjLTYyMmYtNDc0OS1hYzUzLTg0NDAwYTJiYzIwMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyN1QxMDMyMjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lZjUzNmNlMzJmYjY4ZWNhZDA5NTgyOGY1Y2IyMDVmNjRhMjIxNDk5MWU3YzdkOTI0ZTNiYjY5ZDI1ODg5YWFhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.KZQDD2PhbQgXq_8P7K78EWkS_Bo1OiMG5cyv8CBati0">
> I am confused as to why because the module does seem to export a default object of type Ansis. 🤔

The problem is in the tsconfig.
If using `Node16` module then TypeScript can't correctly pick up the type of the default export from `index.d.ts`:

```json
"module": "Node16",
"moduleResolution": "Node16",
```

If using `ESNext` module, then TypeScript works fine:

```json
"module": "ESNext",
"moduleResolution": "Node",
```

Reply:
@darkgnotic 

The issue with `strip` property  is fixed in `v3.5.2`.
The example works fine with `"module": "Node16"` too:
```ts
import ansis from 'ansis';

ansis.strip(ansiText);
```

Reply:
Thank you @webdiscus !